Sync/Synchronizers/Bookmarks/BookmarksRepairRequestor.swift

```diff
@@ -236,19 +248,32 @@ class BookmarksRepairRequestor {
                 var extra = [
                     "flowID": self.flowID,
                     "numIDs": String(self.currentIDs.count),
-                    ]
+                ]
                 if abortReason != nil {
                     extra["reason"] = abortReason
                 }
-                // TODO: TELEMETRY
-                // this.service.recordTelemetryEvent("repair", method, undefined, extra)
-                // reset our state
+
+                let event = Event(category: "sync", method: "repair", object: method, extra: extra)
+                self.recordTelemetry(event: event)
+
                 self.prefs.clearAll()
             }
             return deferMaybe(true)
         }
     }
 
+    func recordTelemetry(event: Event) {
```

Comment:
I suppose it could but it uses a pretty specific `PrefsSyncEvents` key for now. I would move it out but it's unclear how it would need to work outside of the sync ping so I might just keep it here for now.